ios/RNZoomUs.m

```diff
@@ -239,4 +243,24 @@ - (void)onMeetingError:(MobileRTCMeetError)errorCode message:(NSString *)message
   meetingPromiseReject = nil;
 }
 
+- (void)onClickShareScreen:(UIViewController *)parentVC {
+  if (@available(iOS 12.0, *)) {
+    CGRect frame = parentVC.view.bounds;
+    RPSystemBroadcastPickerView *pickerView = [[RPSystemBroadcastPickerView alloc] initWithFrame:frame];
+    pickerView.preferredExtension = screenShareExtension;
+    SEL buttonPressed = NSSelectorFromString(@"buttonPressed:");
+    if ([pickerView respondsToSelector:buttonPressed]) {
+      [pickerView performSelector:buttonPressed withObject:nil];
+    }
+  }
+}
+
+- (BOOL)respondsToSelector:(SEL)aSelector
+{
+    if (aSelector == @selector(onClickShareScreen:)) {
```

Comment:
If no `screenShareExtension` has been configured, we will report to Zoom that our `MobileRTCMeetingServiceDelegate` doesn't support `onClickShareScreen:`.